homeassistant/components/binary_sensor/deconz.py

```diff
@@ -26,10 +27,17 @@
     entities = []
 
     for sensor in sensors.values():
-        if sensor and sensor.type in DECONZ_BINARY_SENSOR:
+        if sensor.type in DECONZ_BINARY_SENSOR:
             entities.append(DeconzBinarySensor(sensor))
     async_add_devices(entities, True)
 
+    async def async_new_sensor(sensor):
```

Comment:
No need to make `async`, just make it a normal function and mark it as `@callback`